internal/output/output.go

```diff
@@ -52,3 +65,25 @@ func Print(c Config, t string, v interface{}) error {
 	}
 	return nil
 }
+
+func templateValue(c Config, defaultTemplate string) (string, error) {
+	value := defaultTemplate
+	templateFormat := ""
+	for _, format := range templateFormats {
+		format += "="
+		if strings.HasPrefix(c.Output(), format) {
```

Comment:
[q] I am not sure what this piece of code does 😖

Reply:
if Output starts with any of the valid templates ("go-template=" or "go-template-file=") then we grab the part that we can use as a template or a file